src/ui/osx/TogglDesktop/Features/Timer/TimerViewController/TimerViewModel.swift

```diff
@@ -231,6 +231,21 @@ final class TimerViewModel: NSObject {
         actionsUsedBeforeStart.insert(TimerEditActionTypeProject)
     }
 
+    func didCreateNewProject(_ project: Project) {
+        if !isRunning {
+            // when TE is running, the project is set on Library level
+            clearProject()
+        }
+
+        timeEntry.projectGUID = project.guid
+        timeEntry.projectColor = project.colorHex
+        timeEntry.projectLabel = project.name
+        timeEntry.taskLabel = project.taskName
+        timeEntry.clientLabel = project.clientName
+
+        onProjectUpdated?(Project(timeEntry: timeEntry))
```

Comment:
just wondering, what's the difference here between using `project` and `Project(timeEntry: timeEntry))`?

Reply:
no difference 🙈 Updated 👍